frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/model/computeRelated.js

```diff
@@ -103,7 +106,22 @@ export const addDiagramDetails = (resourceStatuses, resourceMap, isClusterGroupe
         continue
       }
       //look for pod template hash and remove it from the name if there
-      const { nameNoHash, deployableName } = getNameWithoutPodHash(relatedKindList[i])
+      let { nameNoHash, deployableName } = getNameWithoutPodHash(relatedKindList[i])
+
```

Comment:
Do we want to add a check to see if relatedKindList[i] exist?

Reply:
Thanks @chenz4027 ! I don't think we need this check because this is part of a for loop. If `relatedKindList` is empty then it will just exit the loop.
<img width="761" alt="image" src="https://github.com/user-attachments/assets/0f3bc65b-3e8e-406e-a59f-035c72f23445">